membership-common/src/main/scala/com/gu/memsub/images/ResponsiveImage.scala

```diff
@@ -1,14 +1,6 @@
 package com.gu.memsub.images
 import io.lemonlabs.uri.Uri
-import io.lemonlabs.uri.dsl._
-
-object ResponsiveImageGenerator {
-  def apply(id: String, sizes: Seq[Int], extension: String = "jpg"): Seq[ResponsiveImage] = {
-    sizes.map { size =>
-      ResponsiveImage(id.split("/").fold("https://media.guim.co.uk")({ case (a, b) => a / b }) / s"$size.$extension", size)
```

Comment:
didn't compile due to "/" and "https://media.guim.co.uk" needing to be `Url`s because that's what `/` returns.  Once I realised nothing uses that apart from the tests, I just deleted it.